packages/compass-saved-aggregations-queries/src/components/aggregations-queries-list.tsx

```diff
@@ -198,14 +207,15 @@ const AggregationsQueriesList = ({
       }}
     >
       <VirtualGrid
-        data-testid="my-queries-grid"
+        data-testid="my-queries-list"
         itemMinWidth={CARD_WIDTH}
         itemHeight={CARD_HEIGHT + spacing[2]}
         itemsCount={sortedItems.length}
         renderItem={renderItem}
         itemKey={(index: number) => sortedItems[index].id}
         renderHeader={GridControls}
         headerHeight={spacing[5] + 36}
+        renderEmptyList={<NoSearchResults />}
```

Comment:
thought: Isn't it just `sortedItems.length === 0 ? <NoSearchResults /> : <VirtualGrid ... />`? 

Doing it in`<VirtualGrid>` is totally fine and seems aligned with the design there but is becoming a sort of God component.

Reply:
we can do that. as we render grid controls (filters and sort) in the `VirtualGrid` component, then it will not be visible and user will have no way to clear the filters. 

however, what we can do is render `VirtualGird` first and then conditionally render `NoSearchResults`, but it will require lots of style changes (@gribnoysup suggested this approach). seems more complicated.

Reply:
Ah i understand no is perfect then